hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -845,6 +846,11 @@ bool MDMParser::init(DevStatus* status)
     sendFormated("AT+CIMI\r\n");
     if (RESP_OK != waitFinalResp(_cbString, _dev.imsi))
         goto failure;
+    // Reformat the operator string to be numeric
+    // (allows the capture of `mcc` and `mnc`)
+    sendFormated("AT+COPS=3,2\r\n");
+    if (RESP_OK != waitFinalResp())
```

Comment:
This should have the proper timeout value `if (RESP_OK != waitFinalResp(nullptr, nullptr, COPS_TIMEOUT))`

Reply:
updated